tools/runners/moore.py

```diff
@@ -0,0 +1,49 @@
+import re
+from BaseRunner import BaseRunner
+
+
+class moore(BaseRunner):
+    def __init__(self):
+        super().__init__("moore", "moore")
+
+        self.url = "http://llhd.io/"
+
+    def prepare_run_cb(self, tmp_dir, params):
+        self.cmd = [self.executable]
+
+        for incdir in params['incdirs']:
+            self.cmd.append('-I')
+            self.cmd.append(incdir)
+
+        if params['top_module'] != '':
+            self.cmd.append('-e')
+            self.cmd.append(params['top_module'])
+        else:
+            regex = re.compile(r'module\s+(\w+)\s*[#(;]')
+            for fn in params['files']:
+                with open(fn) as f:
+                    m = regex.search(f.read())
+                    if m:
+                        self.cmd.append('-e')
+                        self.cmd.append(m.group(1))
+
+        self.cmd += params['files']
+
+    def run(self, tmp_dir, params):
+        blacklist = [
+            "std2017/p220.sv",
+            "std2017/p221.sv",
+            "std2017/p745.sv",
+            "std2017/p341.sv",
+            "std2017/p371.sv",
+            "std2017/p759.sv",
+            "std2017/p773.sv",
+        ]
+        for arg in params['files']:
+            for bl in blacklist:
+                if bl in arg:
+                    return ("Skipping blacklisted " + arg, 1)
```

Comment:
I think that this should be moved to the test generator [config file](https://github.com/SymbiFlow/sv-tests/blob/master/conf/generators/meta-path/hdlconvertor.json) to blacklist those tests for all runners.
Just like it was done before in similar cases where other runners misbehaved on some imported tests.

Reply:
Sounds good, I will do that!